Sources/Containerization/ContainerManager.swift

```diff
@@ -192,13 +192,37 @@ public struct ContainerManager: Sendable {
         }
     }
 
-    /// Create a new manager with the provided kernel and initfs mount.
+    /// Create a new manager with the provided kernel, initfs mount, image store
+    /// and optional network implementation.
     public init(
         kernel: Kernel,
         initfs: Mount,
+        imageStore: ImageStore,
```

Comment:
It does not look like I have an `init()` that functions as today, just giving me a default.  Do you think this init() should have imagestore as optional, and if it is `nil`, then assigned the default?

Reply:
I'm not following the first sentence, but yea we probably could have imagestore as optional, I'm trying to remember why I did it this way

Reply:
That is the one below this, albeit it takes in a url.

Reply:
I provided two ways of supplying a "store" as the path is used to store more than just images iirc (on phone or I'd check 😂). One being a url and we'll make an imagestore for you based on the url, or the other being to supply an ImageStore yourself if you want. This would allow you to supply the ImageStore's second argument in its constructor if that was helpful.

Reply:
If you just leave the third arg blank (as it is today) the optional url constructor will "win" and we'll just use the default ImageStore.